<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookReferDialog</class>
 <widget class="QDialog" name="AddBookReferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>تعديل العزو</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>الاسم:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>العزو:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelHelp">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body dir=&quot;rtl&quot; style=&quot;direction:rtl;&quot;&gt;
&lt;p&gt;سيتم تغيير العبارات التالية:&lt;/p&gt;
&lt;a href=&quot;#nass&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;*النص*&lt;/span&gt;&lt;/a&gt; النص الذي تم تحديده&lt;br/&gt;
&lt;a href=&quot;#book&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;*الكتاب*&lt;/span&gt;&lt;/a&gt; عنوان الكتاب&lt;br/&gt;
&lt;a href=&quot;#author&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;*المؤلف* &lt;/span&gt;&lt;/a&gt; اسم المؤلف&lt;br/&gt;
&lt;a href=&quot;#page&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;*الصفحة*&lt;/span&gt;&lt;/a&gt; رقم الصفحة&lt;br/&gt;
&lt;a href=&quot;#part&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;*الجزء*&lt;/span&gt;&lt;/a&gt; رقم الجزء&lt;br/&gt;
&lt;a href=&quot;#haddit&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;*الحديث*&lt;/span&gt;&lt;/a&gt; رقم الحديث&lt;br/&gt;
&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>الغاء</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>حفظ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddBookReferDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>7</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddBookReferDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
